[llvm-amdgpu] Cannot Find `tensorflow/c/c_api.h`

With the most recent update to llvm-amdgpu, I am getting the following error during compilation:

```
FAILED: lib/Analysis/CMakeFiles/LLVMAnalysis.dir/TFUtils.cpp.o 
/bin/c++ \
  -D__STDC_CONSTANT_MACROS \
  -D__STDC_FORMAT_MACROS \
  -D__STDC_LIMIT_MACROS \
  -D_FORTIFY_SOURCE=2 \
  -D_GNU_SOURCE \
  -DGTEST_HAS_RTTI=0 \
  -DNDEBUG  -fno-exceptions \
  -fdata-sections \
  -fdiagnostics-color \
  -ffunction-sections \
  -fno-plt \
  -fno-rtti \
  -fPIC \
  -fvisibility-inlines-hidden \
  -I/include \
  -I/usr/include/libxml2 \
  -Iinclude \
  -Ilib/Analysis \
  -Illvm-project-rocm-4.0.1/llvm/include \
  -Illvm-project-rocm-4.0.1/llvm/lib/Analysis \
  -march=x86-64 \
  -mtune=native \
  -O2 \
  -O3 \
  -pedantic \
  -pipe \
  -std=c++14 \
  -Wall \
  -Wcast-qual \
  -Wdelete-non-virtual-dtor \
  -Werror=date-time \
  -Wextra \
  -Wimplicit-fallthrough \
  -Wno-class-memaccess \
  -Wno-comment \
  -Wno-long-long \
  -Wno-maybe-uninitialized \
  -Wno-missing-field-initializers \
  -Wno-noexcept-type \
  -Wno-redundant-move \
  -Wno-unused-parameter \
  -Wsuggest-override \
  -Wwrite-strings \
  -MD \
  -MT lib/Analysis/CMakeFiles/LLVMAnalysis.dir/TFUtils.cpp.o \
  -MF lib/Analysis/CMakeFiles/LLVMAnalysis.dir/TFUtils.cpp.o.d \
  -o lib/Analysis/CMakeFiles/LLVMAnalysis.dir/TFUtils.cpp.o \
  -c llvm-project-rocm-4.0.1/llvm/lib/Analysis/TFUtils.cpp
llvm-project-rocm-4.0.1/llvm/lib/Analysis/TFUtils.cpp:23:10: fatal error: tensorflow/c/c_api.h: No such file or directory
   23 | #include "tensorflow/c/c_api.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

From looking at the includes specified, there is `-I/include` (which isn't even a valid path) and there isn't the expected `/usr/include/tensorflow` (which contains `/usr/include/tensorflow/tensorflow/c/c_api.h`).

Note that this tensorflow header is owned by tensorflow, which is not a dependency of `llvm-amdgpu`.  Perhaps it should be added as such, either as a regular dependency or an optional dependency?

--- a/llvm-amdgpu/PKGBUILD
+++ b/llvm-amdgpu/PKGBUILD
@@ -14,17 +14,24 @@ source=("${pkgname}-${pkgver}.tar.gz::$url/archive/rocm-$pkgver.tar.gz")
 sha256sums=('8003ede5a1462f46f0bb1b8b16c14ac6e884d322561235f7111ef7c3e74c6b07')
 _dirname="$(basename "$url")-$(basename "${source[0]}" .tar.gz)"
 
+prepare() {
+    cmake -GNinja \
+        -Wno-dev \
+        -S "$_dirname/llvm" \
+        -DCMAKE_INSTALL_PREFIX='/opt/rocm/llvm' \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DLLVM_HOST_TRIPLE=$CHOST \
+        -DLLVM_BUILD_UTILS=ON \
+        -DLLVM_ENABLE_BINDINGS=OFF \
+        -DLLVM_ENABLE_OCAMLDOC=OFF \
+        -DLLVM_ENABLE_PROJECTS='llvm;clang;compiler-rt;lld' \
+        -DLLVM_TARGETS_TO_BUILD='AMDGPU;X86' \
+        -DOCAMLFIND=NO \
+        -DCMAKE_CXX_FLAGS='-I/usr/include/tensorflow' \
+        -DTENSORFLOW_AOT_PATH='/usr/lib/python3.9/site-packages/tensorflow'
+}
+
 build() {
-    cmake -GNinja -Wno-dev -S "$_dirname/llvm" \
-          -DCMAKE_INSTALL_PREFIX='/opt/rocm/llvm' \
-          -DCMAKE_BUILD_TYPE=Release \
-          -DLLVM_HOST_TRIPLE=$CHOST \
-          -DLLVM_BUILD_UTILS=ON \
-          -DLLVM_ENABLE_BINDINGS=OFF \
-          -DLLVM_ENABLE_OCAMLDOC=OFF \
-          -DLLVM_ENABLE_PROJECTS='llvm;clang;compiler-rt;lld' \
-          -DLLVM_TARGETS_TO_BUILD='AMDGPU;X86' \
-          -DOCAMLFIND=NO
     ninja
 }
 

--- a/llvm-amdgpu/PKGBUILD
+++ b/llvm-amdgpu/PKGBUILD
@@ -13,8 +13,9 @@ makedepends=(cmake python ninja)
 source=("${pkgname}-${pkgver}.tar.gz::$url/archive/rocm-$pkgver.tar.gz")
 sha256sums=('8003ede5a1462f46f0bb1b8b16c14ac6e884d322561235f7111ef7c3e74c6b07')
 _dirname="$(basename "$url")-$(basename "${source[0]}" .tar.gz)"
+_pythonver=$(python --version | grep -o "[[:digit:]].[[:digit:]]")
 
-prepare() {
+build() {
     cmake -GNinja \
         -Wno-dev \
         -S "$_dirname/llvm" \
@@ -28,10 +29,7 @@ prepare() {
         -DLLVM_TARGETS_TO_BUILD='AMDGPU;X86' \
         -DOCAMLFIND=NO \
         -DCMAKE_CXX_FLAGS='-I/usr/include/tensorflow' \
-        -DTENSORFLOW_AOT_PATH='/usr/lib/python3.9/site-packages/tensorflow'
-}
-
-build() {
+        -DTENSORFLOW_AOT_PATH='/usr/lib/python${_pythonver}/site-packages/tensorflow'
     ninja
 }
 

